gef-ui/server/controllers/application-details/index.js

```diff
@@ -123,18 +126,27 @@ const stateToPartial = (status, url) => {
   const partials = {
     'review-decision': 'review-decision',
     'confirm-cover-start-date': 'confirm-cover-start-date',
+    'ukef-cover-start-date': 'ukef-cover-start-date',
   };
 
-  return req in partials
-    ? partials[req]
+  return url in partials
+    ? partials[url]
     : template[status];
 };
 
 const applicationDetails = async (req, res, next) => {
   const {
-    params: { applicationId },
+    params: { applicationId, facilityId },
     session: { user, userToken },
   } = req;
+  const facilitiesPartials = [
+    'confirm-cover-start-date',
+    'ukef-cover-start-date',
+  ];
```

Comment:
Is `ukef-cover-start-date` the form submission page? I'm wondering if we should rename?

Reply:
Done